array-manipulation-3/src/equal.js

```diff
@@ -0,0 +1,21 @@
+/* exported equal */
+// set if statements for two empty arrays and if first array length ! == second array length
+// for loop through first array and check if each array's items match up at index i
+
+function equal(first, second) {
+
+  if (first.length === 0 && second.length === 0) {
+    return true;
+  }
+  if (first.length !== second.length) {
```

Comment:
If you checked this first, then the check on line 7 could just look at one of the arguments.
It doesn't matter, just another way to think about it.